sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -129,12 +145,29 @@ export class GCNotifyActionsService {
     const notificationsIds = await this.notificationService.saveNotifications(
       notificationsToSend,
       auditUserId,
-      entityManager,
+      options?.entityManager,
     );
 
-    // TODO: Temporary code to be removed once queue/schedulers are in place.
-    for (const id of notificationsIds) {
-      await this.notificationService.sendEmailNotification(id);
+    if (!options?.notificationsDelayed) {
+      // Execute all the notifications in parallel and return the promisees
+      // to allow the method to await them all.
+      const notificationPromisees = notificationsIds.map((notificationId) =>
+        // Not intended to be used for large volume of notifications. If a large
+        // volume is expected, the notificationsDelayed must be set to true
```

Comment:
I don't mind about it but as it has been pointed out in one of my other PRs, I noticed the comment is missing the period punctuation mark here. 😄

Reply:
Yes, it is valid to point it out. To add some context why it is needed, in the past we started to try to add more comments and the sentences were hard to follow sometimes because there was no upper case, commas, or punctuation at a level that we need to read (or at least I need to read) many times the comments to try to "decode" the idea, which almost invalidates the comment sometimes.
The general idea is simple, try to follow regular sentences in which the period is part of it.
So, are the periods really important, not that much, but they are in the middle of a paragraph.
To keep it easy, a general rule, it must look like an English sentence 😉 

This particular one was addressed already.